system/jhtree/ctfile.cpp

```diff
@@ -1138,6 +1146,18 @@ size32_t CJHTreeBlobNode::getBlobData(unsigned offset, void *dst) const
     return sizeHere;
 }
 
+void CJHTreeRawDataNode::unpack(const void *nodeData, bool needCopy)
+{
+    CJHTreeNode::unpack(nodeData, needCopy);
+    // MORE - what about needCopy flag?
+    const byte *data = ((const byte *) nodeData) + sizeof(hdr);
+    unsigned short len = *reinterpret_cast<const unsigned short *>(data);
```

Comment:
expanded size is only < 64K.  That seems a bit surprising, but less so when I realised the data isn't compressed.